src/index.html

```diff
@@ -18,10 +18,11 @@
     <!-- Favicon Setting -->
     <link rel="icon" type="image/x-icon" href="assets/icons/favicon.ico">
     <meta name="theme-color" content="#ffffff">
-    
     <link href="assets/fonts/lato/fontsLato.css" rel="stylesheet">
-
     <link href="assets/css/noscript-styles.css" rel="stylesheet"/>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet"/>
```

Comment:
@arsen1106 To protect our customer's privacy we must keep external requests to the minimum, and specially avoid big companies known for their tracking practices. Could we find a solution without using Google Fonts?